website/docs/tutorials/guides/testing.md

```diff
@@ -233,10 +233,10 @@ If you are familiar with [Foundry](https://book.getfoundry.sh/forge/cheatcodes.h
 ### `mock_call`
 
 ```python
-def mock_call(contract_address: int, fn_name: str, ret_data: List[int]) -> None: ...
+def mock_call(contract_address: int, fn_name: str, ret_data: List[int]) -> Callable: ...
 ```
 
-Mocks all calls to function with the name `fn_name` of a contract with an address `contract_address` unless [`clear_mock_call`](#clear_mock_call) is used. Mocked call returns data provided in `ret_data`.
+Mocks all calls to function with the name `fn_name` of a contract with an address `contract_address` until the returned callable is called. Mocked call returns data provided in `ret_data`. Mock works globally, for all of the contracts, not only the testing contract.
```

Comment:
```suggestion
Mocks all calls to function with the name `fn_name` of a contract with an address `contract_address` until the returned callable is called. Mocked call returns data provided in `ret_data`. The `mock_call`  will affect calls from contracts deployed by `deploy_contract` cheatcode.
```

Maybe we should remove this information and point out which cheatcodes won't work with `deploy_contract` instead.

Reply:
Not sure about that, I will l leave it